app/jobs/mfa_usage_stats_job.rb

```diff
@@ -0,0 +1,15 @@
+class MfaUsageStatsJob < ApplicationJob
+  queue_as :default
+
+  def perform
+    non_mfa_users = User.where(mfa_level: 0).where.not(id: WebauthnCredential.select(:user_id)).count
```

Comment:
What about to use `User.mfa_disabled` and `User.not_mfa_disabled` `enum` generated scopes?

Reply:
Good point. 

Using `mfa_level: :disabled` worked, but I had no luck with `mfa_level: :not_disabled`. I'll push another commit.